### PR TITLE
STENCIL-3000 adding Changelog file

### PR DESCRIPTION
**Pull request by mcampa**
@bigcommerce/stencil-team

### CODE REVIEW

**Comment by mcampa** [comment]
For future PRs:

We are now using `CHANGELOG.md`. Please add an entry with your change under `draft`

```
## Draft

- <here>

## 1.5.1

- Fix an issue with a horizontal scroll bar showing in theme editor
- Hide Gift Certificates when the setting is disabled in the control panel
- Fix an issue with a close button on the quick search modal on mobile

```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1 +1,5 @@
-Passing validation
\ No newline at end of file
+## 1.5.1
+
+- Fix an issue with a horizontal scroll bar showing in theme editor
```
Comment (mattolson) [inline CHANGELOG.md]: What do you think of adding a PR# and author name for each entry?

Comment (mcampa) [inline CHANGELOG.md]: Sure, do you have an idea of the format?

**Comment by mcampa** [comment]
I deleted the zip file. It was added by mistake on this pr https://github.com/bigcommerce/stencil/pull/915

File: CHANGELOG.md (hunk 1 above)
Comment (mcampa) [inline CHANGELOG.md]: I think the pr link is enough. The link will tell you the author

**Comment by mcampa** [comment]
I'm going to make `stencil release` to change `## draft` for the verstion and date, and post the changes in the github release notes :)

**Comment by mattolson** [comment]
👍

**Pull request merged**